### PR TITLE
Pearl Group Adopter

### PR DESCRIPTION
**Pull request by pkoliskins**
Dear Kyma Community,

Referencing our presentation to Kyma and SAP teams, on integrating our CXify Loyalty application with Kyma for exposing and registering Async and Open APIs, we have received invitation from Marco Dorn to register ourselves under the Adopter' s section.

Best regards,
Pavels

### CODE REVIEW

**Comment by magicmatatjahu** [comment]
@pkoliskins Hi! Please sign CLA, and then I will trigger CI jobs. Thanks! :)

File: content/adopters/adopters.yaml [hunk 1]
```diff
@@ -46,3 +46,8 @@ adopters:
     url: "https://softicious.com"
     logo: "./logos/softicious.svg"  
     content: "Kyma enables us to extend and customize a unique customer experience."
+  - websiteUrl: "https://www.pearlgroup.no/en/"
+    company: "Pearl Group"
+    url: "https://www.pearlgroup.no/en/technology/"
+    logo: "./logos/pearlgroup.svg"
+    content: "Kyma helps our customers to bring rapid innovations into their C/4 HANA and S/4 HANA ecosystems, through our CXify Loyalty application template."
```
Comment (bszwarc) [inline content/adopters/adopters.yaml:53]:
```suggestion
    content: "Kyma helps our customers to bring rapid innovations into their C/4 HANA and S/4 HANA ecosystems through our CXify Loyalty application template."
```

Comment (kazydek) [inline content/adopters/adopters.yaml:53]:
```suggestion
    content: "Kyma helps our customers to bring rapid innovations into their C/4 HANA and S/4 HANA ecosystems through our CXify Loyalty application template."
```

Comment (majakurcius) [reply content/adopters/adopters.yaml:53]: image

**Pull request merged**